packages/server/src/api/app/bsky/util/feed.ts

```diff
@@ -35,8 +35,7 @@ export enum FeedAlgorithm {
   ReverseChronological = 'reverse-chronological',
 }
 
-type FeedItem = GetAuthorFeed.AppBskyGetAuthorFeedFeedItem &
-  GetHomeFeed.AppBskyGetHomeFeedFeedItem
+type FeedItem = GetAuthorFeed.FeedItem & GetHomeFeed.FeedItem
```

Comment:
💎 this looks so much nicer and more intuitive.

Reply:
Yeah, really clutch